test/external-classes.js

```diff
@@ -0,0 +1,22 @@
+import { setup } from './_helpers.js';
+import { describe, expect, test } from 'vitest';
+
+setup({ externalizeClasses: true, externalClasses: ["mt-32", "mb-32"] });
+
+describe('external classes', () => {
+  test('externaliized classes of mt-32 and mb-32 should be excluded from the css', async ({ uno }) => {
```

Comment:
Suuuperminor nitpick
```suggestion
  test('externalized classes of mt-32 and mb-32 should be excluded from the css', async ({ uno }) => {
```